Source/engine.cpp

```diff
@@ -2088,234 +2088,202 @@ void engine_draw_pixel(int sx, int sy)
 // 52B99C: using guessed type int gbNotInView;
 // 69CF0C: using guessed type int gpBufEnd;
 
-/*
- * Xiaolin Wu's anti-aliased line algorithm

```

Comment:
I would leave that comment as otherwise reader loses the original name / authorship reference (even if it is partly mentioned in the referenced page).

Reply:
Well it's not technically anti-aliased algorithm I think since it doesn't uses transparency. Also if you read the description it may have been improved by the other person, I could add the link to the book to the source then maybe?

Reply:
It's up to @galaxyhaxz to decide.  I think mentioning the book (ISBN) would be very nice.

Thanks, I missed description from the book.  Now I get why the original is called `symwuline()` =)